<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditAtomPropertiesDialog</class>
 <widget class="QDialog" name="EditAtomPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Atom Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="set_as_layout">
     <item>
      <spacer name="set_as_spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="set_as_lbl">
       <property name="text">
        <string>Set as:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="set_as_spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="set_as_atom_rb">
       <property name="text">
        <string>Atom</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="set_as_spacer3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="set_as_query_rb">
       <property name="text">
        <string>Query</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="set_as_spacer4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="atom_query_stacked_wdg">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="edit_atom_page">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="atom_element_layout">
         <item>
          <widget class="QLabel" name="element_lbl">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Element:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="element_le">
           <property name="text">
            <string>C</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="schrodinger::sketcher::CommonAtomPropertiesWidget" name="atom_common_props_wdg" native="true"/>
       </item>
       <item>
        <spacer name="edit_atom_spacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="edit_query_page">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTabWidget" name="edit_query_tab_wdg">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="general_tab">
          <attribute name="title">
           <string>General</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>12</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>12</number>
           </property>
           <property name="bottomMargin">
            <number>12</number>
           </property>
           <item>
            <layout class="QGridLayout" name="query_element_layout">
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="query_type_lbl">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Type:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <layout class="QHBoxLayout" name="query_type_layout">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QComboBox" name="query_type_combo"/>
               </item>
               <item>
                <widget class="schrodinger::sketcher::ToolButtonWithPopup" name="periodic_table_btn">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../sketcher.qrc">
                   <normaloff>:/icons/periodic_table.svg</normaloff>:/icons/periodic_table.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>52</width>
                   <height>26</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="1">
              <layout class="QVBoxLayout" name="type_layout">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QComboBox" name="wildcard_combo"/>
               </item>
               <item>
                <widget class="QLineEdit" name="element_list_le">
                 <property name="placeholderText">
                  <string>(comma-separated list)</string>
                 </property>
                 <property name="clearButtonEnabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="specific_element_le">
                 <property name="placeholderText">
                  <string>(element)</string>
                 </property>
                 <property name="clearButtonEnabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="rgroup_sb">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::UpDownArrows</enum>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>999</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="schrodinger::sketcher::CommonAtomPropertiesWidget" name="query_common_props_wdg" native="true"/>
           </item>
           <item>
            <spacer name="general_spacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="other_tab">
          <attribute name="title">
           <string>Advanced</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>12</number>
           </property>
           <property name="rightMargin">
            <number>12</number>
           </property>
           <property name="bottomMargin">
            <number>12</number>
           </property>
           <item row="3" column="1">
            <widget class="QComboBox" name="ring_count_combo">
             <item>
              <property name="text">
               <string>(any)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>&gt;0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>exactly</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="aromaticity_lbl">
             <property name="text">
              <string>Aromaticity (a/A):</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="num_connections_lbl">
             <property name="text">
              <string># Connections (X):</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="smallest_ring_size_lbl">
             <property name="text">
              <string>Smallest ring size (r):</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="total_H_lbl">
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Total H (H):</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="ring_count_lbl">
             <property name="text">
              <string>Ring Count (R):</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <spacer name="other_spacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="ring_bond_count_lbl">
             <property name="text">
              <string>Ring bond count (rb):</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="smarts_query_lbl">
             <property name="text">
              <string>SMARTS query:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QSpinBox" name="ring_bond_count_sb">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="ring_bond_count_combo">
             <item>
              <property name="text">
               <string>(any)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>exactly</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QSpinBox" name="ring_count_sb">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QComboBox" name="aromaticity_combo">
             <item>
              <property name="text">
               <string>(any)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Aromatic (a)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Aliphatic (A)</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="num_connections_le">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="total_H_le">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="6" column="1" colspan="2">
            <widget class="QLineEdit" name="smallest_ring_size_le">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="7" column="1" colspan="2">
            <widget class="QLineEdit" name="smarts_query_le">
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>schrodinger::sketcher::CommonAtomPropertiesWidget</class>
   <extends>QWidget</extends>
   <header>schrodinger/sketcher/edit_atom_properties.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>schrodinger::sketcher::ToolButtonWithPopup</class>
   <extends>QToolButton</extends>
   <header>schrodinger/sketcher/tool_button_with_popup.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../sketcher.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditAtomPropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditAtomPropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
